.rubocop.yml

```diff
@@ -14,7 +14,6 @@ AllCops:
   - "**/Rakefile"
   - pkg/**/*
   - spec/fixtures/**/*
-  - spec/**/*
```

Comment:
@sheenaajay it looks like this caused rubocop to yell about cop failures in /spec/defines/download_spec.rb.